ait/dsn/bin/server.py

```diff
@@ -0,0 +1,275 @@
+            ait.core.log.error("RAF instance is not started. Unable to stop.")
+            bottle.response.status = 400
+        else:
+            self.rcf_instance.stop()
+            self.rcf_started = False
+
+    def cltu_stop_handler(self):
+        if (not self.cltu_started):
+            ait.core.log.error("CLTU instance is not started. Unable to stop.")
+            bottle.response.status = 400
+        else:
+            self.cltu_instance.stop()
+            self.cltu_started = False
+
+    def raf_unbind_handler(self):
+        if (not self.raf_bound):
+            ait.core.log.error("RAF instance is not bound. Unable to unbind.")
+            bottle.response.status = 400
+        else:
+            self.raf_instance.unbind()
+            self.raf_bound = False
+
+    def rcf_unbind_handler(self):
+        if (not self.rcf_bound):
+            ait.core.log.error("RCF instance is not bound. Unable to unbind.")
+            bottle.response.status = 400
+        else:
+            self.rcf_instance.unbind()
+            self.rcf_bound = False
+
+    def cltu_unbind_handler(self):
+        if (not self.cltu_bound):
+            ait.core.log.error("CLTU instance is not bound. Unable to unbind.")
+            bottle.response.status = 400
+        else:
+            self.cltu_instance.unbind()
+            self.cltu_bound = False
+
+    def raf_disconnect_handler(self):
+        if (not self.raf_connected):
+            ait.core.log.error("RAF instance is not connected. Unable to disconnect.")
+            bottle.response.status = 400
+        else:
+            self.raf_instance.disconnect()
+            self.raf_connected = False
+
+    def rcf_disconnect_handler(self):
+        if (not self.rcf_connected):
+            ait.core.log.error("RCF instance is not connected. Unable to disconnect.")
+            bottle.response.status = 400
+        else:
+            self.rcf_instance.disconnect()
+            self.rcf_connected = False
+
+    def cltu_disconnect_handler(self):
+        if (not self.cltu_connected):
+            ait.core.log.error("CLTU instance is not connected. Unable to disconnect.")
+            bottle.response.status = 400
+        else:
+            self.cltu_instance.disconnect()
+            self.cltu_connected = False
+
+    def cltu_upload_handler(self):
+        if (not self.cltu_started):
+            ait.core.log.error("CLTU instance is not started. Unable to upload CLTU.")
+            bottle.response.status = 400
+        else:
+            wsock = bottle.request.environ.get('wsgi.websocket')
+            if not wsock:
+                ait.core.log.error("Expected WebSocket request")
+                bottle.abort(400, 'Expected WebSocket request.')
+            while True:
+                try:
+                    tc_data = wsock.receive()
+                    self.cltu_instance.upload_cltu(tc_data)
+                except geventwebsocket.WebSocketError:
+                    break
```

Comment:
I think this needs rethought. Users shouldn't be hitting an endpoint to upload a CLTU